polyglot.js

```diff
@@ -71,29 +104,58 @@ class PolyGlot {
 			actors.push(game.user.character);
 		for (let actor of actors) {
 			try {
-				// Don't duplicate the value in case it's a not an array
-				for (let lang of actor.data.data.traits.languages.value)
-				    this.known_languages.add(lang)
-			} catch (err) { 
+				switch(game.system.id.toLowerCase()){
+					default:
+						// Don't duplicate the value in case it's a not an array
+						for (let lang of actor.data.data.traits.languages.value)
+								this.known_languages.add(lang)
+						break;
+					case "wfrp4e":
+						let re = /Language \((.+)\)/i;
+						for (let item of actor.data.items){
+							let found = item.name.match(re);
+							if(found) {
+								// adding only the descriptive language name, not "Language (XYZ)"
+								this.known_languages.add(found[1]);
+							}
+						}
+						break;
+				}
+			} catch (err) {
 				// Maybe not dnd5e, pf1 or pf2e or corrupted actor data?
 			}
 		}
 		if (this.known_languages.size == 0) {
 			if (game.user.isGM)
 				this.known_languages = new Set(Object.keys(PolyGlot.languages))
-			else
-				this.known_languages.add("common");
+			else {
+				switch(game.system.id.toLowerCase()){
+					default:
+						this.known_languages.add("common");
+						break;
+					case "wfrp4e":
+						this.known_languages.add("Reikspiel");
+				}
```

Comment:
I would rather have a `getDefaultLanguage` here instead. which would do that. I would say having a switch/case here makes sense, but again, default needs to be last, and I would rather see specific cases for all systems, i.e: 
```js
case "wfrp4e":
	this.known_languages.add("Reikspiel");
        break;
case "dnd5e":
case "pf1":
case "pf2e":
default:
     this.known_languages.add("common");
     break;
}
```
(I just made the mistake of not having a `break` for wfrp4e because I copy/pasted it to move it up. I noticed as I was reviewing my review comments... This is why I say to always add `break`, even if it's the last case, or even if there's a return before it.)